example/main.py

```diff
@@ -164,10 +184,26 @@ def webhook():
         raise ValueError("FB_VERIFY_TOKEN does not match.")
     elif request.method == "POST":
         messenger.handle(request.get_json(force=True))
-        print(request.get_json(force=True))
-        print(
-            request.get_json(force=True)["entry"][0]["messaging"][0]["message"]["text"]
-        )
+        payload = request.get_json(force=True)
+        if jmespath.search("entry[0].messaging[0].sender.id", payload) is not None:
+            post_back_event = [
+                {
+                    "user_id": jmespath.search(
+                        "entry[0].messaging[0].sender.id", payload
+                    ),
+                    "url": jmespath.search(
+                        "entry[0].messaging[0].postback.payload", payload
+                    ),
+                }
+            ]
+            postback_filename = "postback.json"
+            if not Path(postback_filename).exists():
+                post_back_events = []
+            else:
+                post_back_events = json.load(open(postback_filename, "r"))
+            post_back_events.extend(post_back_event)
+            json.dump(post_back_events, open(postback_filename, "w"))
+            return ""
```

Comment:
2. once user hit `Like` button, fb would send a payload like this

```json
{
    "object": "page",
    "entry": [
        {
            "id": "160712400714277",
            "time": 1647157758983,
            "messaging": [
                {
                    "sender": {
                        "id": "1413683625375061"
                    },
                    "recipient": {
                        "id": "160712400714277"
                    },
                    "timestamp": 1647157758626,
                    "postback": {
                        "title": "Like",
                        "payload": "https://www.youtube.com/watch?v=S_1WBzXFyBs&t=3752s",
                        "mid": "m_vd8GvKqga9DEwz4_AX4RmILBtrrgUaBxwObkV7tvm_TDeZiIcuNNrlwq965jeyzHuOAH2yCnykRPsbz4w0cPLQ"
                    }
                }
            ]
        }
    ]
}
```

in this block, we parse this payload and then save it into json file

Reply:
3. future work for you @linbeta 😛 
    * beta to refactor this function to obey `SRP` (single responsibility principle) 
    * save these events into BigQuery (our warehouse)

Reply:
Wow! this is cool, got it!
Now I learned how to handle the postback. I guess the SRP part would be a little bit challenging for me. Will let you know if there's any blocker. XD